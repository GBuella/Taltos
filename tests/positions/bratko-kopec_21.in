force
nopost
setboard 3rn2k/ppb2rpp/2ppqp2/5N2/2P1P3/1P5Q/PB3PPP/3RR1K1 w - -
nps 100000
st 200
search_sync
hash_value_min 190
