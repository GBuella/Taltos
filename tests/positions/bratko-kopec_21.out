1. Nh6
ok
